eng/scripts/KillProcesses.ps1

```diff
@@ -10,6 +10,12 @@ function _kill($processName) {
     }
 }
 
+function _killSeleniumJavaInstances(){
```

Comment:
nit: This name isn't quite accurate, it's killing our java instances, which might be selenium. Don't block on it though.